libs/datasets/sources/covid_county_data.py

```diff
@@ -40,10 +44,88 @@ class CovidCountyDataDataSource(data_source.DataSource):
         }
     }
 
+    @classmethod
+    def synthesize_test_metrics(cls, data: pd.DataFrame) -> Tuple[pd.DataFrame, pd.Series]:
```

Comment:
A bit of a larger question, but should this belong in covid data public or covid data model? After reading through this a bit more, I would expect it to be in covid-data-public - mostly in the spirit of keeping the data source wrappers pretty lightweight.

Reply:
With the idea that maybe we want all of cases, total tests, pos tests, neg tests in the model I think it make sense to keep this transformation here. I see what happens in covid-data-public as pretty bare-bones transformation to the common format with very little business logic that we may be subject to experimentation. stuff in this repo may need more careful consideration. the line isn't between the two isn't very sharp to me right now so there is a bit of 'ship whatever is easiest'